spring-boot-examples/new-customer.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we avoid these payloads to be in source control?

Reply:
done